go.mod

```diff
@@ -18,9 +19,9 @@ require (
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
+	github.com/zzxwill/stern v1.13.3-alpha
```

Comment:
please use https://github.com/oam-dev/stern, and keep the old lib like this : https://github.com/rancher/rio/blob/master/go.mod#L18